commerce-frontend-js/src/main/resources/META-INF/resources/js/dataset_display/DatasetDisplay.es.js

```diff
@@ -0,0 +1,153 @@
+import {ClayIconSpriteContext} from '@clayui/icon';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, {useState, useRef} from 'react';
+
+import DataSetDisplayContext from './DataSetDisplayContext.es';
+import ManagementBar from './management_bar/index.es';
+import Pagination from './pagination/index.es';
+import Table from './table/Table.es';
+
+// function getApiEndpoint(dataSetAPI, pageSize, currentPage = 1, filters = []) {
+// 	const formattedFilters = Object.keys(filters)
+// 		.map(filter => {
+// 			return `${encodeURIComponent(filter)}=${encodeURIComponent(
+// 				filters[el]
+// 			)}`;
+// 		})
+// 		.join('&');
+
+// 	const endpoint = `${dataSetAPI}&pageSize=${pageSize}&page=${currentPage}&p_auth=${Liferay.authToken}&${formattedFilters}`;
+
+// 	return endpoint;
+// }
+
+function DataSetDisplay(props) {
```

Comment:
DataSetDisplay or DatasetDisplay, let's be consistent with the file name /cc @ethanbustad